### ISSUE TITLE
File paths are compiled incorrectly during batch imports

### ISSUE THREAD

**Issue by davidmcclure** [scholarslab/Neatline#196]
When a batch of Omeka items is imported into an exhibit by way of the "Import Items" form, paths to files in the compiled metadata output (eg, images) **point to the local filesystem, not the web-accessible location of the files**.

For example, if I have an Omeka item with an image file attached to it, and I import the item via "Import Items," the `Neatline_ImportItems` process create markup like this:

``` html
<a class="download-file" href="http:///Applications/MAMP/htdocs/tei/application/scripts/files/original/0abfa685e9fa0cdfcae8e3e9b3f6a8b0.png">sample-image.png</a>
```

As soon as I save the item individually from the editor (which causes the item to be re-compiled - this time in the context of a regular request, as opposed to a background job), the correct path is compiled:

``` html
<a class="download-file" href="http://localhost:8888/tei/files/original/0abfa685e9fa0cdfcae8e3e9b3f6a8b0.png">neatline-2.0-alpha1-small.png</a>
```

In other words, it seems that there's some kind of environment variable (perhaps `WEB_ROOT`?) that's not set to the correct value in the context of the background process.

**Comment by davidmcclure** [comment]
Fixed.
